### ISSUE TITLE
Bug: typo in the control-gate matrices on the gates tutorial

### ISSUE THREAD

**Issue by vandieren** [Qiskit/qiskit-tutorials#1392]
### Where does the issue happen?

https://qiskit.org/documentation/tutorials/circuits/3_summary_of_quantum_operations.html#Two-qubit-gates

### What is the content issue?

The indented block of mathematical notation (CU: |0>...) after "Therefore, under the action of CU, the basis vectors are transformed according to" matches the block further down on the page, but these should be different from each other.  The second block appears to be correct, and the first block needs to be edited.

**Comment by Guillermo-Mijares-Vilarino** [comment]
Hi @vandieren, let's take a look at those blocks.

Let's start with the first one (I included some text to avoid any confusion about which block is which):

![Captura de pantalla 2023-01-27 a las 15 45 18](https://user-images.githubusercontent.com/106545082/215114643-714e87bf-68eb-447b-b61f-10bec9c53dc9.png)

In this case the $C_U$ is controlled by qubit $0$, so if that qubit is in the state $|1\rangle$, the gate $U$ will get applied to qubit $1$. Otherwise, the state doesn't change at all.

In the first and third rows of that block, we can see that qubit 0 is in the state $|0\rangle$ so there's no change.
However, for the second and fourth rows qubit $0$ is in the state $|1\rangle$. Let's see if the gate $U$ is applied to qubit $1$: 
* In the second row the initial state is $|0\rangle_1 \otimes |1\rangle_0$ and it gets changed to $U|0\rangle_1 \otimes |1\rangle_0 = \big(U|0\rangle\big)_1 \otimes |1\rangle_0 $, that is, qubit $1$ gets changed from $|0\rangle$ to $U|0\rangle$. In other words, the gate $U$ gets applied to qubit $1$ and qubit $0$ is left alone, exactly as intended.
* Similarly for the fourth row we start with the state $|1\rangle_1 \otimes |1\rangle_0$ and it changes to $U|1\rangle_1 \otimes |1\rangle_0 = \big(U|1\rangle\big)_1 \otimes |1\rangle_0$ so qubit $1$ goes from $|1\rangle$ to $U|1\rangle$, that is, the gate $U$ is applied to this qubit and qubit $0$ doesn't change. So, again, the $U$ gate is applied to qubit $1$ as intended.

Therefore, the first block seems to be fine.

Let's look at the second one:
![Captura de pantalla 2023-01-27 a las 15 45 42](https://user-images.githubusercontent.com/106545082/215114730-f36848d5-0f0a-492f-810c-48d44787aa12.png)

Here the control qubit of the $C_U$ is qubit $1$, so if that qubit is in the state $|1\rangle$ the gate $U$ will get applied to qubit $0$, otherwise there would be no change.

Notice that for the first and second rows qubit $1$ is in state $|0\rangle$, so the state stays the same.
However for the third and fourth rows the state of qubit $1$ is $|1\rangle$. Let's check them individually:
* In the third row the initial state is $|1\rangle_1 \otimes |0\rangle_0$ and it becomes $|1\rangle_1 \otimes U|0\rangle_0 = |1\rangle_1 \otimes \big(U|0\rangle\big)_0$, that is, qubit $0$ goes from $|0\rangle$ to $U|0\rangle$ and qubit $1$ stays at $|1\rangle$, so the gate $U$ is applied to qubit $0$ as expected.
*  Similarly, in the fourth row we start with the state $|1\rangle_1 \otimes |1\rangle_0$ and it's changed to $|1\rangle_1 \otimes U|1\rangle_0 = |1\rangle\_1 \otimes \big(U|1\rangle\big)_0$ so qubit $0$ goes from $|1\rangle$ to $U|1\rangle$, that is, the gate $U$ is applied to this qubit and qubit $1$ doesn't change. So the $U$ gate is applied to qubit $0$ as intended.

**In short, both blocks are different and both are correct.**

I hope you found this explanation helpful 😄

**Comment by vandieren** [comment]
it appears correct now.  thanks

**Comment by Guillermo-Mijares-Vilarino** [comment]
Now that it's solved I'm closing the issue
